src/Fragments/RandomFragment.ts

```
@@ -30,25 +30,33 @@ export default class RandomFragment extends FragmentBase {
 
-    // 48 bit parts due to randomInt range limitation
+    // 47 bit parts due to randomInt range limitation
     // max param limit of Number.MAX_SAFE_INTEGER
-    let rndNum = BigInt(1);
-    for (let i = 1; i < this.bits / 48; i += 1) {
-      rndNum *= BigInt(randomInt(1, 1 << 48));
+    let rndNum = BigInt(0);
+    for (let i = 0; i < this.bits; i += 47) {
+      rndNum |=
+        BigInt(
+          randomInt(0, Number(1n << BigInt(Math.min(this.bits - i, 47)))),
```

ESLint

[error] Replace `⏎··········randomInt(0,·Number(1n·<<·BigInt(Math.min(this.bits·-·i,·47)))),` with `randomInt(0,·Number(1n·<<·BigInt(Math.min(this.bits·-·i,·47)))))·<<`

(prettier/prettier)